market/connector/google-geo-api/google-geo-api-demo/src_hd/google/getp/map/map.xhtml

```diff
@@ -0,0 +1,33 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:f="http://xmlns.jcp.org/jsf/core"
+	xmlns:h="http://xmlns.jcp.org/jsf/html"
+	xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+	xmlns:ic="http://ivyteam.ch/jsf/component"
+	xmlns:p="http://primefaces.org/ui"
+	xmlns:pe="http://primefaces.org/ui/extensions">
+<h:body>
+	<ui:composition template="/layouts/basic-8.xhtml">
```

Comment:
use the 'frame-8.xhtml' for dialogs: so that they fit naturally into the latest Portal without breaking the layout

Reply:
- [x] use the 'frame-8.xhtml' for dialogs: so that they fit naturally into the latest Portal without breaking the layout